src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/HealthCertificateAccess.swift

```diff
@@ -0,0 +1,49 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import base45_swift
+
+public struct HealthCertificateAccess {
+
+    // MARK: - Public
+
+    public func extractCBORWebTokenHeader(base45: String) -> Result<CBORWebTokenHeader, HealthCertificateDecodingError> {
```

Comment:
> base45: String

I would suggest a `typealias Base45 = String` to distinguish encoded strings from normal ones.

Reply:
Similar to the CBOR data (and potentially some other types in the app as well): `typealias CBORData = Data`

Reply:
🤩 Like it! Added!